css/theme.less

```diff
@@ -211,6 +211,10 @@ body:before {
     margin: 40px -40px 0;
     width: 800px;
     background-color: @ini_background_alt;
+
+    bdi:first-child {
```

Comment:
Minor: Because this is a hack, it could really benefit from a comment in the code that explains what it is doing.

Reply:
Comment added